### PR TITLE
MM-12460 Fill the Theme field for team members on export

### PR DESCRIPTION
**Pull request by noxer**
#### Summary

Fills the "Theme" property for team members in bulk exports.

#### Ticket Link

Fixes https://github.com/mattermost/mattermost-server/issues/9541

### CODE REVIEW

File: app/export.go [hunk 1]
```diff
@@ -249,10 +249,10 @@ func (a *App) ExportAllUsers(writer io.Writer) *model.AppError {
 	return nil
 }
 
-func (a *App) buildUserTeamAndChannelMemberships(userId string) (*[]UserTeamImportData, *model.AppError) {
+func (a *App) buildUserTeamAndChannelMemberships(userID string) (*[]UserTeamImportData, *model.AppError) {
```
Comment (jespino) [inline app/export.go]: We are using `userId` everywhere, we prefer to stick with that until we change everything at the same time. Do you mind to revert that part of the change?

Comment (noxer) [reply app/export.go]: Yes, sorry. It is such an automatism for me at this point. Makes sense to keep the code base consistent.

File: app/export.go [hunk 2]
```diff
@@ -269,11 +269,17 @@ func (a *App) buildUserTeamAndChannelMemberships(userId string) (*[]UserTeamImpo
...
 
 		// Do the Channel Memberships.
-		channelMembers, err := a.buildUserChannelMemberships(userId, member.TeamId)
+		channelMembers, err := a.buildUserChannelMemberships(userID, member.TeamId)
 		if err != nil {
 			return nil, err
 		}
 
+		// Get the user theme
+		themePreference, err := a.Srv.Store.Preference().Get(member.UserId, model.PREFERENCE_CATEGORY_THEME, "")
```
Comment (jespino) [inline app/export.go]: You are getting here the main theme preference, probably you want to pass the "teamId" in the last parameter to get the Theme preference for the team

Comment (noxer) [reply app/export.go]: Ah didn't know about that. Thanks.

**Comment by noxer** [comment]
I've rebased the PR...

**Comment by grundleborg** [comment]
@noxer would you be able to merge `master` into this PR branch and then we can go ahead and merge it.

**Comment by noxer** [comment]
Thank you for the review, I've merged the current master branch.

**Pull request merged**